c2rust-ast-exporter/build.rs

```diff
@@ -119,6 +119,12 @@ fn build_native(llvm_info: &LLVMInfo) {
             println!("cargo:rustc-link-search=native={}", libdir);
         }
         _ => {
+            if env::var("DOCS_RS").is_ok() {
```

Comment:
Could we just skip `build_native()` entirely if this env var is set? That would be simpler.

Reply:
Yeah, does pretty much the same thing.  Should it go at the top of `build_native` (I'd put it here) or around the call to `build_native`?

Reply:
I'd prefer outside of build_native, to indicate that we should not even try to build in this docs mode.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/482/commits/50ab8099512ce1c93bca801e053d2839e2d86860.